source/includes/dev/_microservice_dev_guide.md

```diff
@@ -141,6 +140,14 @@ Test execution requires roughly the same env vars as running the actual microser
  
 See right for how to run tests for microservices in different languages. You can also review the `entrypoint.sh` file content, under the `test` section, which will contain the exact command you need to execute.
 
+#### Common errors and pitfalls
```

Comment:
Love this section!

Reply:
well show it some love by trying to remember if you ran into something that's not covered here :P Kidding aside, I struggled a bit to write it, as I don't run into these pitfalls anymore - I already have the whole API configured on my computer. Since you went through this process recently, if you have any suggestions of problems you faced, let me know.